report/docker_run.sh

```diff
@@ -112,12 +167,19 @@ $PYTHON run_all_experiments.py \
   --num-samples 10 \
   --delay "${DELAY:?}" \
   --context \
+  --introspector-endpoint ${INTROSPECTOR_ENDPOINT} \
   --model "$MODEL"
 
 export ret_val=$?
 
 touch /experiment_ended
 
+if [[ "$USE_LOCAL_INTROSPECTOR" = "true" ]]
+then
+  echo "Shutting down introspector"
+  curl --silent http://localhost:8080/api/shutdown || true
+fi
+
```

Comment:
Maybe move to the same new file as `launch_local_introspector_deployment`?

Reply:
We can't -- this has to be called after the experiment so we should call it from the one that controls the experiment running

Reply:
Would this trick work?
https://github.com/google/oss-fuzz-gen/blob/main/report/upload_report.sh#L102-L106

E.g.,
```bash
while [[ ! -f /experiment_ended ]]; do
  sleep 600
done

# Stop server now.
```